examples/reactrouter-example/vite.config.ts

```diff
@@ -0,0 +1,6 @@
+import { reactRouter } from '@react-router/dev/vite'
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+  plugins: [reactRouter()],
```

Comment:
Are you sure we are running in framework mode without defining this file?

https://github.com/remix-run/react-router-templates/blob/main/default/react-router.config.ts

Are the loaders/actions client-side only without this?